apps/designer/app/designer/features/style-panel/style-source-input/style-source.tsx

```diff
@@ -39,7 +41,75 @@ const menuCssVars = ({ show }: { show: boolean }) => ({
   [menuTriggerVisibilityVar]: show ? "visible" : "hidden",
 });
 
+const MenuTrigger = styled("button", baseButtonStyle, {
```

Comment:
Not sure it's a good idea to import `baseButtonStyle` from button. Now every time someone needs to make changes to button, they would also have to test whether they broke this component

Reply:
alright, removed